apps/vscode/src/app/workspace-tree/get-task-execution-schema.ts

```diff
@@ -119,12 +119,15 @@ export async function getTaskExecutionSchema(
             }
 
             schematic.options.forEach(s => {
+              // TODO: mixup between items and enum has been a source for recent bugs,
+              //  util.ts normalizeSchema sets items from enum.
+              //  Should this getProjectEntries be over there?
```

Comment:
This fixes the projects dropdown, but the mixup between items and enum is still a bit confusing.